entrypoint/restartservice/restartservice_test.go

```diff
@@ -101,8 +101,8 @@ var _ = Describe("Test:", func() {
 		})
 	}
 
-	It("Displays Cluster details", func() {
-		testcase.TestDisplayClusterDetails()
+	It("Displays Cluster details after tests", func() {
+		testcase.DisplayClusterDetails()
```

Comment:
1. Is this change related to this PR? If not its unnecessary for the reviewer to chase this change in the PR context.
2. Why do we need DisplayClusterDetails, why not call the underlying methods instead?

Reply:
actually this should not live here since its not a test, not a specific test helper.

should live on shared and not also be used as a `It` spec, since its not a actual test or feat

Reply:
Updated